patterns/1-initial/innersource-contractor-model-terms.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+InnerSource Contractor Model Terms
+
+## Patlet
+
+Contract developers are often not motivated to engage in InnerSource activities. This patlet includes some terms which could be included in their initial contract to ensure engagement in the InnerSource process.
+
+## Problem
+
+Contractor developers are often not motivated (through forces described below) to not engage in InnerSource activities. Once delivered, and even if the code is made visible, their projects are often less likely to be part of successful InnerSourced engagements.
+
+## Context
+
+This problem exists where an organization either:
+
+- Out-sources the development of a well defined project or
+- Engagages external firms for staff augmentation and has mixed teams of permanent employees with a large percentage of contract staff.
+
+## Forces
+
+Contractor Motivation and Constraints:
+
+- Often contracts with third party developers are very focused on delivering an end result in the fastest possible fashion. As a result, all InnerSource activities (e.g. responding to third party PRs) are considered to be distractions or something that will “slow down” ultimate delivery.
```

Comment:
I believe it goes beyond this. There are two main forms of contract in the consulting world: Fixed-Price (FP) and Time-and-Materials (T&M). With FP, project profitability is _directly_ impacted by the number of hours worked. Contractors are incentivized to work as few hours as possible to achieve the outcomes stated in the SOW. T&M is more flexible, as there are usually no defined outcomes. Contracting orgs love FP with well defined scopes. Clients usually like a mix depending on the outcome (staff augmentation vs wanting a particular deliverable, for example).